ODLA/platforms/dnnl/odla_dnnl_binary.cc

```diff
@@ -132,9 +132,7 @@ static void binary_eltwise_T(alg_binary_eltwise alg, void* dst,
       out = in_l.pow(in_r);
       break;
     }
-    default: {
-      assert(0);
-    }
+    default: { assert(0); }
```

Comment:
why change odla_dnnl and odla_popart?

Reply:
lint automatically changing it. Reverted now.